rdt/__init__.py

```diff
@@ -94,82 +87,3 @@ def get_demo(num_rows=5):
             'dollars_spent': dollars_spent
         })
     ], ignore_index=True)
-
-
-def _get_addon_target(addon_path_name):
-    """Find the target object for the add-on.
-
-    Args:
-        addon_path_name (str):
-            The add-on's name. The add-on's name should be the full path of valid Python
-            identifiers (i.e. importable.module:object.attr).
-
-    Returns:
-        tuple:
-            * object:
-                The base module or object the add-on should be added to.
-            * str:
-                The name the add-on should be added to under the module or object.
-    """
-    module_path, _, object_path = addon_path_name.partition(':')
-    module_path = module_path.split('.')
-
-    if module_path[0] != __name__:
-        msg = f"expected base module to be '{__name__}', found '{module_path[0]}'"
-        raise AttributeError(msg)
-
-    target_base = sys.modules[__name__]
-    for submodule in module_path[1:-1]:
-        target_base = getattr(target_base, submodule)
-
-    addon_name = module_path[-1]
-    if object_path:
-        if len(module_path) > 1 and not hasattr(target_base, module_path[-1]):
-            msg = f"cannot add '{object_path}' to unknown submodule '{'.'.join(module_path)}'"
-            raise AttributeError(msg)
-
-        if len(module_path) > 1:
-            target_base = getattr(target_base, module_path[-1])
-
-        split_object = object_path.split('.')
-        addon_name = split_object[-1]
-
-        if len(split_object) > 1:
-            target_base = attrgetter('.'.join(split_object[:-1]))(target_base)
-
-    return target_base, addon_name
-
-
-def _find_addons():
-    """Find and load all RDT add-ons.
-
-    If the add-on is a module, we add it both to the target module and to
-    ``system.modules`` so that they can be imported from the top of a file as follows:
-
-    from top_module.addon_module import x
-    """
-    group = 'rdt_modules'
-    for entry_point in iter_entry_points(group=group):
-        try:
-            addon = entry_point.load()
-        except Exception:  # pylint: disable=broad-exception-caught
-            msg = f'Failed to load "{entry_point.name}" from "{entry_point.module_name}".'
-            warnings.warn(msg)
-            continue
-
-        try:
-            addon_target, addon_name = _get_addon_target(entry_point.name)
-        except AttributeError as error:
-            msg = f"Failed to set '{entry_point.name}': {error}."
-            warnings.warn(msg)
-            continue
-
-        if isinstance(addon, ModuleType):
-            addon_module_name = f'{addon_target.__name__}.{addon_name}'
-            if addon_module_name not in sys.modules:
-                sys.modules[addon_module_name] = addon
-
-        setattr(addon_target, addon_name, addon)
-
-
-_find_addons()
```

Comment:
This code has to stay! 😅 Otherwise enterprise addons won't work

Reply:
Good catch, thanks! Updated in [f332d2d](https://github.com/sdv-dev/RDT/pull/766/commits/f332d2de8b98a4c2e491008b6aed7f629aeeaff2)